scripts/statcast_timing.py

```diff
@@ -0,0 +1,30 @@
+import sys
+import time
+from pybaseball import statcast
+
+EXPECTED_TIME_THRESHOLD = 30
+START_DATE = "2018-08-01"
+END_DATE = "2018-08-10"
+
+def main():
+    if len(sys.argv) == 2:
+        time_threshold = float(sys.argv[1])
+    else:
+        time_threshold = EXPECTED_TIME_THRESHOLD
+
+    start_time = time.time()
+    _ = statcast(START_DATE, END_DATE)
+    end_time = time.time()
+    query_time = end_time-start_time
+    if query_time > time_threshold:
```

Comment:
Do you know if subprocess works in CI? We could run this in a subprocess with a timeout and kill if it went too far over (let's say over a minute).

Reply:
no I don't know. the goal is to make CI run faster? it feels like solving a problem we don't have.

Reply:
@schorrm I looked up the [documentation on this](https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions#jobsjob_idtimeout-minutes) and implemented a 1 minute timeout on the large-query step